api/lib/mno_enterprise/concerns/controllers/jpi/v1/teams_controller.rb

```diff
@@ -30,6 +30,8 @@ def create
     authorize! :manage_teams, parent_organization
     @team = parent_organization.teams.create(team_params)
 
+    MnoEnterprise::EventLogger.info('team_add', current_user.id, 'Team created', parent_organization, {team_name: @team.name}) if @team
```

Comment:
`{team_name: @team,name}` could be moved to the model `to_audit_event` as well as the `organization_id`?
So we'd have
```ruby
MnoEnterprise::EventLogger.info('team_add', current_user.id, 'Team created', @team) if @team
```